activities.js

```diff
@@ -4,7 +4,7 @@ var os = require("os");
 
 var async = require("async");
 
-var swfr = require("./index");
+var worker = require("./index").Worker;
```

Comment:
Renaming `swfr` => `Worker` and `Worker` to `WorkerWritable`.  Let me know if you have a better naming scheme, I just needed to shuffle names around so the exports could export both this functionality and the Decider.